### PR TITLE
Workflows for version_bump and build_release.

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ip_allocation_range: 172.30.0.0/16
 weave_kube_image_tag: 2.8.1
-weave_npc_image_tag: 2.8.1
+weave_npc_image_tag: 2.8.1
```
Comment (MartinSchmidt) [inline chart/values.yaml:3]: - [ ] missing empty line

File: .github/workflows/bump_version.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEFAULT_BUMP: none
+        WITH_V: true
+        RELEASE_BRANCHES: main
+        INITIAL_VERSION: 0.0.0
+        DRY_RUN: false
+        TAG_CONTEXT: repo
+        PRERELEASE_SUFFIX: beta
+        VERBOSE: true
```
Comment (MartinSchmidt) [inline .github/workflows/bump_version.yaml:25]: - [ ] missing empty line

File: .github/workflows/build_release.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    types: 
+      - completed
+  push:
+    tags:
+      - 'v[0-9]\.[0-9]\.[0-9]*'
+
+jobs:
+  build:
+    # Only run the job if the Version Bump action was succesfull
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```
Comment (MartinSchmidt) [inline .github/workflows/build_release.yml:16]: does it still run if a version tag is set manually?

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]

File: .github/workflows/build_release.yml (hunk 3 above)
Comment (jacobcrawford) [reply .github/workflows/build_release.yml]: Bug is fixed and pushed to wiki

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**